python/tk_framework_desktopserver/shotgun/scripts/get_commands.py

```diff
@@ -235,6 +229,9 @@ def cache(
     finally:
         sys.path = original_sys_path
 
+    # Now that we have sgtk.util loaded, use it to make sure we have only utf-8 data in the args
+    arg_data = sgtk.util.json.loads(json.dumps(arg_data))
```

Comment:
My feeling here is that it would be better to use `sgtk.util.unicode.ensure_contains_str` on the `arg_data` since we've already had to load it using the standard library, rather than dumping and loading again.

Reply:
https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/util/json.py#L96

Reply:
That makes perfect sense. I made the change